### ISSUE TITLE
Remove RandomSplitter from examples

### ISSUE THREAD

**Issue by ai-fast-track** [airctic/icevision#431]
## 📓 Documentation Update
RandomSplitter is now passed by default. Therefore, we need to update the examples, in the [examples folder](https://github.com/airctic/icevision/tree/master/examples).

1- The following line should be removed:
```
# Randomly split our data into train/valid
data_splitter = RandomSplitter([0.8, 0.2])
```

2- The following line should be modified: remove `data_splitter` from the `parser.parse()` call
**Before:**
```
train_records, valid_records = parser.parse(data_splitter)
```

**After:**
```
train_records, valid_records = parser.parse()
```

**Comment by singhalpranav22** [comment]
I have submitted a pull request for the same.

**Comment by ai-fast-track** [comment]
Closed: Implemented in #435
